lib/LaTeXML/Post/MathML.pm

```diff
@@ -774,8 +774,8 @@ sub stylizeContent {
       $size = undef; }
     # If requested relative size, and in script or scriptscript, we'll need to adjust the size
     elsif (($size =~ /%$/) && ($LaTeXML::MathML::STYLE =~ /script/)) {
-      my $req = $size;                               $req =~ s/%$//;
-      my $ex  = $stylesize{$LaTeXML::MathML::STYLE}; $ex  =~ s/%$//;
+      my $req = $size; $req =~ s/%$//;
+      my $ex = $stylesize{$LaTeXML::MathML::STYLE}; $ex =~ s/%$//;
```

Comment:
I am tempted to ask if you ran `tools/latexmllint` on all of your files, but I am confident you did. The spaces changing back and forth may be another sign that we're using different versions of perltidy, I should likely sync up with yours.

Reply:
yeah, I did, using the same tidyrc as in tools. But I have also noticed that things bounce around for no obvious reason.  I have perltidy version v20181120, fwiw.

Reply:
```
$ perltidy -v
This is perltidy, v20200619
```

Well, that's quite the difference... I guess I am too much on the bleeding edge again :clock1: